lib/health_check/middleware_health_check.rb

```diff
@@ -7,16 +7,15 @@ def initialize(app)
 
     def call(env)
       uri = env['PATH_INFO']
-      if uri =~ /^\/?#{HealthCheck.uri}\/?([-_0-9a-zA-Z]*)([-_0-9a-zA-Z]*)(\.(\w*))?/
-       checks = $1 + ($1 != '' && $2 != '' ? '_' : '') + $2
-       puts checks
-       checks = 'standard' if checks == ''
-       response_type = $4
-       begin
-         errors = HealthCheck::Utils.process_checks(checks)
-       rescue => e
-         errors = e.message.blank? ? e.class.to_s : e.message.to_s
-       end
+      if uri =~ /^\/?#{HealthCheck.uri}\/?([-_0-9a-zA-Z]*)(\.(\w*))?/
+        checks = $1
+        checks = 'standard' if checks == ''
+        response_type = $3
```

Comment:
`response_type = $3` => `response_type = $3 || 'default_type'`

and below in If statement you can explicitly put if as another if condition or case statement. Benefit of this is that user quickly see relation between $4 and content_type